.github/workflows/lint.yml

```diff
@@ -37,9 +37,9 @@ jobs:
       - name: MegaLinter
         id: ml
         uses: oxsecurity/megalinter@v7.1.0
+        continue-on-error: true
```

Comment:
I thought in this PR https://github.com/hyperledger/identus-cloud-agent/pull/1092 we want to fail this check when there is a linter fix. What is the reason to still `continue-on-error`? :thinking:

Reply:
so, the corner case that is fixed (previously the step 3 was ignored):
1. mega linter finds errors and applies auto-fixes, 
2. as there are errors, the check must be red, 
3. even if the check is red, mega linter creates the PR with fixes

Reply:
Just to confirm, should this be red? https://github.com/hyperledger/identus-cloud-agent/actions/runs/9203500025/job/25315143878